modules/farm_fd2_school/src/entrypoints/Vue2/App.vue

```diff
@@ -81,6 +89,7 @@
     <hr />
 
     <textarea
+      v-model="commentBox"
```

Comment:
Using `commentBox` as the property name here may be a little misleading.  The value here isn't a "Box" but the value in the box.  So maybe `commentText` or just `comment` would be better?